### DISCUSSION TITLE
Online Vueform Builder with Translations

### DISCUSSION THREAD

**Discussion by ridesuite** [vueform/vueform#345] [Questions]
I'm new to Vueform. I have a requirement to build multiple complex forms for a SaaS web application. I do not require to integrate the builder into the product, but use it to reduce the time and effort to 'hand code' all forms. I understand that i18 support is only provided by the paid 'white label' subscription, which is acceptable. Is there any way I can use within your online builder or is this something I need to run locally with an integrated builder? Thanks!

**Comment by adamberecz** [comment]
You’re right—there’s no translation function in the online builder. However, here’s an example that demonstrates how localization works: https://builder.vueform.com/examples/basics-locales

To set it up with specific locales, you’ll need a local version configured to your preferred languages, which you can then use for form building. You might also consider creating a wrapper to add custom save/load methods if needed.

Additionally, we sometimes offer development licenses at a reduced price for users who plan to use Vueform Builder solely for development rather than full integration. If this could be interesting for you, please drop me an email at adam@vueform.com.
